echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/slack/SlackNotificationService.groovy

```diff
@@ -58,7 +59,7 @@ class SlackNotificationService implements NotificationService {
         response = slack.sendCompactMessage(new CompactSlackMessage(text), address, true)
       } else {
         response = slack.sendMessage(
-          new SlackAttachment("Spinnaker Notification", text, (InteractiveActions)notification.interactiveActions),
+          new SlackAttachment(subject, text, (InteractiveActions)notification.interactiveActions),
           address, true)
```

Comment:
Now the subject here will show up!
https://github.com/spinnaker/keel/blob/master/keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/ManualJudgementNotifier.kt#L115

Reply:
Yes, thank you, thank you! I totally forgot about this bug!